src/components/structures/RoomView.js

```diff
@@ -557,7 +546,35 @@ module.exports = React.createClass({
             self.setState({
                 searchInProgress: false
             });
-        });
+        }).done();
```

Comment:
Ignore that, you're calling done after the then which does handle the error. Can we just `s/then/done/` on :505 please?